### ISSUE TITLE
varbinary not returned by Subsonic Query, always set to null

### ISSUE THREAD

**Issue by Kaarel** [subsonic/SubSonic-3.0#105]
I have a column in SQL Server 2005 table of type varbinary(MAX). Generated active record class has the corresponding property type byte[]. I can successfully new-up the active record class representing the table, initialize values and save. The static method Find of the active record class returns the full set of data. When using the SubSonic.Linq.Structure.Query&lt;T&gt; then the byte[] field is always null.

In other words

var query = new Query&lt;MyTable&gt;();
var result = (from x in query select x).ToList();

if MyTable has a property of type byte&#91;&#93; (varbinary(MAX) in SQL Server 2005) then that property is always null in the result.
